wrappers/kn/src/nativeMain/kotlin/zxingcpp/ImageView.kt

```diff
@@ -0,0 +1,52 @@
+package zxingcpp
+
+import cnames.structs.ZXing_ImageView
+import kotlinx.cinterop.*
+import zxingcpp.cinterop.*
+import kotlin.experimental.ExperimentalNativeApi
+import kotlin.native.ref.createCleaner
+
+@OptIn(ExperimentalForeignApi::class)
+class ImageView(
+	val data: ByteArray,
+	val width: Int,
+	val height: Int,
+	val format: ImageFormat,
+	val rowStride: Int = 0,
+	val pixStride: Int = 0,
+) {
+	private val pinnedData = data.pin()
+	val cValue: CPointer<ZXing_ImageView>? =
+		ZXing_ImageView_new(
+			pinnedData.addressOf(0).reinterpret(),
+			width,
+			height,
+			format.rawValue,
+			rowStride,
+			pixStride
+		)
+
+	@Suppress("unused")
+	@OptIn(ExperimentalNativeApi::class)
+	private val cValueCleaner = createCleaner(cValue) { ZXing_ImageView_delete(it) }
+
+	@Suppress("unused")
+	@OptIn(ExperimentalNativeApi::class)
+	private val pinnedDataCleaner = createCleaner(pinnedData) { it.unpin() }
+}
+
+@OptIn(ExperimentalForeignApi::class)
+enum class ImageFormat(internal val rawValue: ZXing_ImageFormat) {
```

Comment:
For consistency with the other enum definition, please call the variable `cValue` instead of `rawValue`.

Reply:
done in https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/57a858a9603b3fc6021221b477632c4572274b1d